lib/streamlit/report_session.py

```diff
@@ -54,30 +54,23 @@ class ReportSession(object):
     """
     Contains session data for a single "user" of an active report
     (that is, a connected browser tab).
-
```

Comment:
I'm curious, did your IDE automatically do these whitespace changes? Is it possible to undo them?

One element of keeping PRs small is to separate out formatting changes from functional (code) changes; it's much faster to review 2 such PRs sequentially, than to combine them both here.